docker-compose.yml

```
@@ -26,14 +23,13 @@ services:
-      # credentials are hardcoded as it is only for development
-      POSTGRES_USER: codesirius
-      POSTGRES_PASSWORD: codesirius
-      POSTGRES_DB: codesirius
+      POSTGRES_USER: ${DB_USER}
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_DB: ${DB_NAME}
     ports:
       - "5432:5432"
     healthcheck:
-      test: [ "CMD-SHELL", "pg_isready -U codesirius" ]
+      test: [ "CMD-SHELL", "pg_isready -U ${DB_USER}" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)